aiidalab_widgets_base/structures.py

```diff
@@ -1009,14 +1049,21 @@ def def_axis_p1(self, _=None):
 
     def def_axis_p2(self, _=None):
```

Comment:
Why is the first argument always ignored? Could we just drop that as part of our operator decorator?

Reply:
> Why is the first argument always ignored?

It is because the button widgets provide this argument when calling the function. It has no useful information, so we ignore it.

> Could we just drop that as part of our operator decorator?

I think the answer depends a bit on how we design the decorator/decorators. So we should probably answer [this](https://github.com/aiidalab/aiidalab-widgets-base/pull/310#discussion_r884730170) one first.

Reply:
Please move forward as you see fit.